h/migrations/versions/08d3c5a8bd08_fix_indices.py

```diff
@@ -0,0 +1,36 @@
+"""Fix indices created on a value instead of a column."""
+
+import sqlalchemy as sa
+from alembic import op
+
+revision = "08d3c5a8bd08"
+down_revision = "e87d20882edb"
+
+
+def upgrade():
+    # Creating a concurrent index does not work inside a transaction
+    op.execute("COMMIT")
+
+    # Drop the old, incorrect indices
+    op.execute("DROP INDEX IF EXISTS ix__user__email")
+    op.execute("DROP INDEX IF EXISTS subs_uri_lower_idx_subscriptions")
```

Comment:
Out of curiosity, what made the old indexes wrong? I cannot tell due to my limited SQLAlchemy knowledge.